packages/reactotron-app/App/Foundation/TimelineHeader.js

```diff
@@ -66,6 +67,7 @@ class TimelineHeader extends Component {
             <div style={Styles.title}>{TITLE}</div>
           </div>
           <div style={Styles.right}>
+            <IconFilter size={Styles.iconSize} style={Styles.toolbarClear} onClick={ui.openFilterTimelineDialog} />
```

Comment:
Probably need to add some padding to the right of this icon or use flexbox to put some space between the icons... they are kinda close